tests/Query/Mysql/DateTest.php

```diff
@@ -36,6 +36,15 @@ public function testDateAdd2()
         $this->assertEquals($sql, $q->getSql());
     }
 
+    public function testFromUnixtime()
+    {
+        $dql = "SELECT p FROM DoctrineExtensions\Tests\Entities\Date p WHERE FROM_UNIXTIME(CURRENT_TIME())";
+        $q = $this->entityManager->createQuery($dql);
+
+       $sql = "SELECT d0_.id AS id0, d0_.created AS created1 FROM Date d0_ WHERE FROM_UNIXTIME(CURRENT_TIME())";
+       $this->assertEquals($sql, $q->getSql());
```

Comment:
or `CURRENT_TIME()` :wink: this query would never make sense, but should pass the test so I'm happy :smile: